xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -675,7 +675,14 @@ The `GetSingle` method throws an exception if the entire UTF-8 encoded token val
       <Docs>
         <summary>Reads the next JSON token value from the source, unescaped, and transcoded as a string.</summary>
         <returns>The token value parsed to a string, or <see langword="null" /> if <see cref="P:System.Text.Json.Utf8JsonReader.TokenType" /> is <see cref="F:System.Text.Json.JsonTokenType.Null" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
```

Comment:
No need to add this since this info is already on the returns tag. I'll remove it.